tck/api/src/main/java/io/astefanutti/metrics/cdi/se/ConcurrentGaugedClassBeanTest.java

```diff
@@ -106,24 +106,24 @@ public void instantiateTest() {
     @Test
     @InSequence(1)
     public void countedMethodsNotCalledYet() {
-        assertThat("Counters are not registered correctly", registry.getConcurrentGauges().keySet(), is(equalTo(counterMIDs)));
-        // Make sure that the counters haven't been incremented
+        assertThat("Concurrent Gauges are not registered correctly", registry.getConcurrentGauges().keySet(), is(equalTo(counterMIDs)));
+        // Make sure that the max-values haven't been incremented
```

Comment:
Why not convert that non-executable comments to meaningful assertion messages instead?

Reply:
Fair point, just did that. This would be applicable to a lot of other test cases as well, but that would be a larger  task to go through the tests and do some cleanup.